docs/narr/router.rst

```diff
@@ -46,16 +46,22 @@ request enters a :app:`Pyramid` application through to the point that
    object.  The former contains a dictionary representing the matched dynamic
    elements of the request's ``PATH_INFO`` value, and the latter contains the
    :class:`~pyramid.interfaces.IRoute` object representing the route which
-   matched.  The root object associated with the route found is also generated:
+   matched.
+   
+   A :class:`~pyramid.events.BeforeTraversal` :term:`event` is sent to any
+   subscribers.
+
+   The root object associated with the route found is also generated:
    if the :term:`route configuration` which matched has an associated
```

Comment:
Next the root object associated with the route found is also generated.
If the...